### PR TITLE
Fixes 1.25

### LINKED ISSUE #1324: Redundant FROMCRS parameter causes WMS loading to fail
https://github.com/hslayers/hslayers-ng/blob/2a3d1f7ccb4d264d3986e80c6d82f015b111d097/components/compositions/compositions-parser.service.ts#L160

Is there a reason to have 'EPSG:3857' hardcoded here? In case of application with different projection, WMS fails to serve GetMap requests if the service does not support 3857.

### LINKED ISSUE #1349: Layman authorization opens in a new window
It should open in a dialog inside the map.

### LINKED ISSUE #1350: Remove only added layers with "Remove all layers" button
The "Remove all layers" icon in the Layer Manager should not remove layers which are defined in the app config file. In case we want it to be removable, the param can be set to true in the config.
Also, there are different labels. The icon is called "Remove all layers", but later in the dialog it says "Remove all added layers". This should be unified.

### LINKED ISSUE #1355: Layer editor opens at wrong place after the page is reloaded
https://localhost:8080/?hs_panel=layermanager&hs_x=1609802.342187116&hs_y=6396453.6057200935&hs_z=10&visible_layers=OpenStreetMap&layerSelected=WMTS

image

### LINKED ISSUE #1359: Composition layers overlay drawing
When loading some compositions from Micka and then drawing new features in the map, those are hidden behind the added layers. Tested with "Covid map 1st September 2020".
image

### LINKED ISSUE #1363: Unsaved drawing gets cleared when saved
After adding some features to the unsaved drawing and then trying to save it to a new layer, those featuers already drawn dissapear.

### PR DESCRIPTION
**Pull request by FilipLeitner**
Fix #1349 
Fix #1350
Partially fixes #1324. Havent deleted from cesium.
Fix #1355
Fix #1363
FIX #1359

### CODE REVIEW

File: components/layermanager/layermanager.service.ts [hunk 1]
```diff
@@ -640,7 +640,7 @@ export class HsLayerManagerService {
   removeAllLayers(): void {
     const to_be_removed = [];
     this.HsMapService.map.getLayers().forEach((lyr) => {
-      if (lyr.get('removable') == undefined || lyr.get('removable') == true) {
+      if (lyr.get('removable') == true) {
```
Comment (raitisbe) [inline components/layermanager/layermanager.service.ts:643]: Could you add a comment so we dont forget how its supposed to work?

**Review by raitisbe — APPROVED** [review]

File: components/layermanager/layermanager.service.ts (hunk 1 above)
Comment (FilipLeitner) [reply components/layermanager/layermanager.service.ts:643]: Yes, of course

File: assets/locales/lv.json [hunk 2]
```diff
@@ -45,7 +45,8 @@
     "thisIsRequired": "Šis ir nepieciešams!",
     "tileSize": "\"Flīžu\" izmērs",
     "useProxy": "Izmantojiet starpniekserveri",
-    "useTiles": "Izmantojiet \"flīzes\""
+    "useTiles": "Izmantojiet \"flīzes\"",
+    "addSelected": "Add selected layers to the map"
```
Comment (jmacura) [inline assets/locales/lv.json:49]: @DailisLangovskis @raitisbe Perhaps one of you could fill in the translation before merge?

File: components/add-layers/wms/add-wms-layer.directive.html [hunk 3]
```diff
@@ -98,9 +98,11 @@
             </tbody>
         </table>
 
-        <button class="btn btn-primary" style="position: sticky;
-        bottom: 1.5rem; float: left;" (click)="selectAllLayers(data.services)">{{'ADDLAYERS.selectAllLayers' | translate}}</button>
-        <button class="btn btn-primary btn-sm" style="position: sticky;
-        bottom: 1.5rem; float: right;" (click)="addLayers(true)"><i class="icon-plus"></i></button>
+
+        <div class="w-100 d-flex justify-content-between bg-light pb-2" style="position: sticky;
```
Comment (jmacura) [inline components/add-layers/wms/add-wms-layer.directive.html:102]: Very pretty, nicely done 🙂

**Review by jmacura — APPROVED** [review]
I have tested what I could and it all work, so if no more commits are coming, it's good to go.
Just note my note about translation 😉

File: assets/locales/lv.json (hunk 2 above)
Comment (raitisbe) [reply assets/locales/lv.json]: @jmacura done

**Pull request merged**